.github/issue_template.md

```diff
@@ -24,6 +24,9 @@ As an [Actor], I want [feature] so that [why].
 
 **Frequency:** [What percentage of the time does it occur?]
 
+### Logs (if a bug)
+Please paste relevant logs from Besu (and the consensus client, if running proof of stake) from before and after the issue. 
```

Comment:
if logs are too big, zip and upload them and include a link?

Reply:
Maybe changing "paste" to "post" would cover that?